ipam/client/backends/phpipam.py

```diff
@@ -218,6 +218,54 @@ def add_top_level_subnet(self, subnet, description):
 
         return True
 
+    def add_subnet(self, subnet, parent_subnet, description):
+        """
+        Add a subnet if can be inserted in parent subnet.
+        """
+        parent_subnet_id = self.find_subnet_id(parent_subnet)
+
+        subnet_len = subnet.prefixlen
+        if not subnet.overlaps(parent_subnet):
```

Comment:
You may also want to check that candidate subnet prefix length < parent subnet one.